docs/kepler_model_server/get_started.md

```diff
@@ -59,12 +73,16 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
 name: kepler-cfm
-namespace: system
+namespace: kepler
 data:
     MODEL_CONFIG: |
-        NODE_COMPONENTS_ESTIMATOR=true
+        MODEL_SERVER_ENABLE: "true"
 ```
 
 See more in [Kepler Power Estimation Deployment](./power_estimation.md)
 
-## Step 4: [Learn how to train the power model and give back to the community](https://github.com/sustainable-computing-io/kepler-model-server/tree/main/model_training)
\ No newline at end of file
+## Step 3: Learn how to train the power model and give back to the community
+
+As you may be aware, it's essential to tailor power models to specific machine types rather than relying on a single universal model. We eagerly welcome contributions from the community to help build alternative power models on your machine through the model server project.
```

Comment:
I would change `universal` to generic.

The work universal gives the impression that the power model covers all possible scenarios.